toonz/sources/toonz/xshcolumnviewer.cpp

```diff
@@ -1823,8 +1958,27 @@ void ColumnArea::mouseMoveEvent(QMouseEvent *event) {
     m_tooltip = tr("Click to select camera");
   else if (o->rect(PredefinedRect::DRAG_LAYER).contains(mouseInCell)) {
     m_tooltip = tr("Click to select column, drag to move it");
+  } else if (o->rect(PredefinedRect::LAYER_NUMBER).contains(mouseInCell)) {
+    if (o->isVerticalTimeline())
+      m_tooltip = tr("Click to select column, drag to move it");
+    else
+      m_tooltip = tr("Click to select column");
+  } else if (o->rect(PredefinedRect::LAYER_NAME).contains(mouseInCell)) {
+    if (o->isVerticalTimeline())
+      m_tooltip =
+          tr("Click to select column, drag to move it, double-click to edit");
+    else if (column && column->getSoundColumn()) {
+      // sound column
+      if (o->rect(PredefinedRect::SOUND_ICON).contains(mouseInCell))
+        m_tooltip = tr("Click to play the soundtrack back");
+      else if (o->rect(PredefinedRect::VOLUME_AREA).contains(mouseInCell))
+        m_tooltip = tr("Set the volume of the soundtrack");
+    } else
+      m_tooltip = tr("Click to select column, double-click to edit");
   } else if (o->rect(PredefinedRect::LOCK_AREA).contains(mouseInCell)) {
     m_tooltip = tr("Lock Toggle");
+  } else if (o->rect(PredefinedRect::CONFIG_AREA).contains(mouseInCell)) {
+    m_tooltip = tr("Additional level settings");
```

Comment:
How about changing this tool tip to "Additional **column** settings" for the purposes of accuracy?

Reply:
I agree. Fixed.